symplyphysics/laws/relativistic/lorentz_transformation_of_time.py

```diff
@@ -1,54 +1,79 @@
+"""
+
 from sympy import Eq, sqrt
-from sympy.physics.units import speed_of_light
 from symplyphysics import (
-    units,
     Quantity,
-    Symbol,
     validate_input,
     validate_output,
+    symbols,
+    clone_as_symbol,
+    quantities,
 )
 from symplyphysics.core.expr_comparisons import expr_equals
 
-# Description
-## Let us consider two inertial reference frames `S` (lab frame) and `S'` (proper frame). The proper frame moves
-## with speed `v` relative to the lab frame. Both frames contain identical fixed (relative to their own frame)
-## clocks that are synchronized according to the [Einstein rule](https://en.wikipedia.org/wiki/Einstein_synchronisation).
-## Let `x, y, z, t` be the coordinates and time of some event in frame `S`, and `x', y', z', t'` be the coordinates
-## and time of the same event in frame `S'`. Assuming that the space is uniform and isotropic and that the time is
-## uniform, there exists a linear dependence between `x, y, z, t` and `x', y', z', t'`, which is called the Lorentz
-## transformation of space and time.
-
-# Law: t' = (t - (V * x) / c**2) / sqrt(1 - (V / c)**2)
-## t' - time in proper (S') reference frame
-## t - time in lab (S) reference frame
-## x - position in lab (S) reference frame
-## V - speed of proper (S') frame relative to lab (S) frame
-## c - speed of light
-
-# Conditions
-## - Space is uniform and isotropic.
-## - Time is uniform.
-## - The relative frame velocity is parallel to the `x` axis.
-
-# Notes
-## - Lab frame S is usually thought as stationary, and proper frame S' is the one that is considered to be moving
-##   relative to lab frame and the moving object in question is at rest in the proper frame.
-## - In this law, the Lorentz transformation from the lab frame `S` into the proper frame `S'` is described. In order to get
-##   an opposite transformation (from the proper frame `S'` into the lab frame `S`), replace all primed variables with unprimed
-##   ones and vice verce, and replace `V` with `-V`. This is consistent with the fact that frame `S` can be viewed as moving
-##   with speed `-V` relative to frame `S'`, and hence the same Lorentz transformation can be applied.
-## - In the limit of `V/c << 1` the formula reduces to the classical Galilean transformation `t' = t`.
-
-# Links: Wikipedia, first formula in box <https://en.wikipedia.org/wiki/Lorentz_transformation#Coordinate_transformation>
-
-time_in_proper_frame = Symbol("time_in_proper_frame", units.time)
-time_in_lab_frame = Symbol("time_in_lab_frame", units.time)
-position_in_lab_frame = Symbol("position_in_lab_frame", units.length)
-proper_frame_speed_in_lab_frame = Symbol("proper_frame_speed_in_lab_frame", units.velocity)
+time_in_proper_frame = clone_as_symbol(symbols.time, display_symbol="t'", display_latex="t'")
+"""
+:symbols:`time` in proper frame :math:`S'`.
+"""
+
+time_in_lab_frame = symbols.time
```

Comment:
It might be a good idea to make naming consistent across laws, i.e. use `position_in_second_frame`, `position_in_first_frame` and so on.